Demo-iOS.xcodeproj/project.pbxproj

```diff
@@ -1043,7 +1075,7 @@
 					"../../iOS/**",
 					"../../Core/**",
 				);
-				IPHONEOS_DEPLOYMENT_TARGET = 5.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
```

Comment:
I'm fine with making iOS8 the minimum, but not in the 2.x branch. That's a big jump, and we should push a tagged release first, and move to e.g. a 3.x branch.

Reply:
This is demo..What's the problem cause min deployment target version ?

Reply:
Ah! Sorry, reading the huge patch I didn't notice that was the demo project - my mistake.

Yep, I have no objection to the demo project being iOS8+ :). Anyone who's making new projects today and targetting less than 8 is ... um ... probably not someone we would support anyway (too expensive/difficult).